github-for-jira.sd.yml

```diff
@@ -242,7 +242,7 @@ workers:
             Lower: 30
             Upper: 80
           Metrics:
-            - Expression: "MAX([cpuUsage, memoryUsage, (pushQueueMessages/250)*100, (backfillQueueMessages/80)*100])"
+            - Expression: "MAX([cpuUsage, memoryUsage, (pushQueueMessages/250)*100, (backfillQueueMessages/20)*100, (branchQueueMessages/250)*100, (deploymentQueueMessages/250)*100])"
```

Comment:
I've decreased threshold for the backfill queue as well, cuz we have less messages there in general